stress/cpuset_manager.go

```diff
@@ -68,7 +69,12 @@ func (manager *cpuStressserManager) TrackInjectorCores(cgroup cgroup.Manager, us
 	// read cpuset allocated cores
 	manager.log.Infow("retrieving target cpuset allocated cores")
 
-	cpusetCores, err := cgroup.Read("cpuset", "cpuset.cpus")
+	cpusetFile := "cpuset.effective_cpus"
+	if cgroup.IsCgroupV2() {
+		cpusetFile = "cpuset.cpus.effective"
```

Comment:
```suggestion
  
		// If “cpuset.cpus” is empty, the “cpuset.cpus.effective” file shows all the CPUs from the parent cgroup 
		// that can be available to be used by this cgroup. Otherwise, it should be a subset of “cpuset.cpus” 
		// unless none of the CPUs listed in “cpuset.cpus” can be granted. In this case, it will be treated just like an empty “cpuset.cpus”.
		cpusetFile = "cpuset.cpus.effective"
```